python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -484,8 +486,12 @@ def tags_for_backfill_id(backfill_id: str) -> Mapping[str, str]:
         return {BACKFILL_ID_TAG: backfill_id}
 
     @staticmethod
-    def tags_for_tick_id(tick_id: str) -> Mapping[str, str]:
-        return {TICK_ID_TAG: tick_id}
+    def tags_for_tick_id(tick_id: str, has_evaluations: bool = False) -> Mapping[str, str]:
+        if has_evaluations:
+            automation_tags = {AUTO_MATERIALIZE_TAG: "true", ASSET_EVALUATION_ID_TAG: tick_id}
```

Comment:
are we going to migrate the AUTO_MATERIALIZE_TAG name?

Reply:
mm yeah should get around to that eventually -- probably will need to handle backcompat for quite awhile so the sooner we make the change the better